src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,41 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int SIZE = 12;
+    private static final int EMPTY = -1;
+    private final int[] numbers = new int[SIZE];
+
+    public int total = EMPTY;
+
+    public int getTotal() {
+        return total;
+    }
+
+    protected void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+   protected boolean callCheck() {
+        return total == EMPTY;
```

Comment:
Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).
This constant should be different (separate) than the one in peekaboo and countOut methods.